kotlinpoet/src/nonJvmMain/kotlin/com/squareup/kotlinpoet/jvm/alias/JvmModifier.nonJvm.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.kotlinpoet.jvm.alias
+
+public actual enum class JvmModifier {
+  PUBLIC,
```

Comment:
Do we actually need to re-declare the values in an actual enum class?

Reply:
It seems that `expect enum class` needs to be re-declare in `actual` if it's not a `typealias`, otherwise you'll get an error:
```
The following declaration is incompatible because some entries from expected enum are missing in the actual enum:
    expect enum class JvmModifier : Enum<JvmModifier>
```